freqtrade/constants.py

```diff
@@ -0,0 +1,121 @@
+# pragma pylint: disable=too-few-public-methods
+
+"""
+List bot constants
+"""
+
+
+class Constants(object):
```

Comment:
Perhaps it would be easier not to define a class for constants and just define them at the module level in constants.py? That way there is no need to instantiate a class every time you want to use a constant but you can use them with

    import constants
    timeout = constants.RETRY_TIMEOUT

Reply:
Some of the class use in this PR seems a bit overkill, I agree.

Reply:
I'm also in favor of defining constants on module level